core/services/relay/evm/write_target.go

```diff
@@ -69,7 +69,8 @@ func NewWriteTarget(ctx context.Context, relayer *Relayer, chain legacyevm.Chain
 			},
 		},
 	}
-	cw, err := NewChainWriterService(lggr.Named("ChainWriter"), chain.Client(), chain.TxManager(), chainWriterConfig)
+
+	cw, err := NewChainWriterService(lggr.Named("ChainWriter"), chain.Client(), chain.TxManager(), nil, chainWriterConfig)
```

Comment:
shouldn't we set the gas estimator properly here?

Reply:
I didn't think that was necessary since Keystone doesn't use `GetFeeComponents` yet.

Reply:
I don't understand why we would leave this out. It is trivial to set both the max gas price and estimator since they are already both available in-scope.
```suggestion
	chainWriterConfig.MaxGasPrice = chain.Config().EVM().GasEstimator().PriceMax()
	cw, err := NewChainWriterService(lggr.Named("ChainWriter"), chain.Client(), chain.TxManager(), chain.GasEstimator(), chainWriterConfig)
```